src/world/world.c

```diff
@@ -24,15 +39,54 @@ static const s32 rodata_73DA0[] = {
 
 INCLUDE_ASM(s32, "world/world", load_map_by_IDs);
 
-INCLUDE_ASM(s32, "world/world", get_current_map_config);
+s32 get_current_map_config() {
+    return D_800A41E0;
+}
 
-INCLUDE_ASM(MapConfig*, "world/world", get_current_map_header);
+MapConfig *get_current_map_header() {
```

Comment:
Nit: our codestyle is to have the asterisk of pointer types hug the left, not the right. i.e. `s32*` not `s32 *`

Reply:
Oh yeah forgot that

Reply:
Fixed.